### PR TITLE
Packaging up log-anomaly-detector-k8s as a python package

### PR DESCRIPTION
**Pull request by zmhassan**
## Background
As we progress in this project some of our components could be good candidates for reuse by other projects.

## Solution
I created a setup.py to automate the installation process. And allow us to package up our project.

### CODE REVIEW

**Comment by zmhassan** [comment]
This work is related to issue #92

**Comment by zmhassan** [comment]
@durandom This PR is ready

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+""" Setup.py for packaging log-anomaly-detector as library """
+import os
+from setuptools import setup
+
+os.system('pip install git+https://github.com/sevamoo/SOMPY.git' +
+          '@76b60ebd6ffd550b0f7faaf632451dfd68827bf7')
+os.system('pip install ' +
+          'git+https://github.com/jaybaird/python-bloomfilter.git' +
+          '@2bbe01ad49965bf759e31781e6820408068862ac')
```
Comment (fridex) [inline setup.py]: Calling pip while pip installs packages seems to be not a good approach. What about directly adding these requirements to `REQURED_PKG` below?

Comment (zmhassan) [reply setup.py]: I'm going to open a git issue for that. Thank you @fridex

**Pull request merged**